src/test/groovy/com/pretz/parkingmanager/api/VehicleStatusMvcSpec.groovy

```diff
@@ -0,0 +1,80 @@
+package com.pretz.parkingmanager.api
+
+import com.pretz.parkingmanager.IntegrationTest
+import com.pretz.parkingmanager.domain.ParkingRate
+import com.pretz.parkingmanager.domain.ParkingSession
+import com.pretz.parkingmanager.repository.ParkingSessionRepository
+import org.junit.experimental.categories.Category
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.web.servlet.MockMvc
+import spock.lang.Specification
+
+import java.sql.Timestamp
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+@Category(IntegrationTest.class)
+class VehicleStatusMvcSpec extends Specification {
+
+    @Autowired
+    MockMvc mockMvc
+
+    @Autowired
+    ParkingSessionRepository parkingSessionRepository
+
+    private String testVehicleId = "TYR2345"
```

Comment:
czy to nie powinna być statyczna stała?

Reply:
Mogła być statyczna, mogła być w setupie, widocznie nie uznałem tej kwestii za aż tak kluczową dopóki parametr jest przekazywany tam gdzie trzeba